src/Tag.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react'
+import propTypes from '@dhis2/prop-types'
+import cx from 'classnames'
+
+import { colors } from './theme.js'
+import { tagVariantPropType } from './common-prop-types.js'
+;('') // TODO: https://github.com/jsdoc/jsdoc/issues/1718
+
+/**
+ * @module
+ * @param {Tag.PropTypes} props
+ * @returns {React.Component}
+ * @example import { Tag } from @dhis2/ui-core
+ * @see Specification: {@link https://github.com/dhis2/design-system/blob/master/atoms/tag.md|Design system}
+ * @see Live demo: {@link /demo/?path=/story/tag--default|Storybook}
+ */
+
+export const Tag = ({
+    neutral,
+    negative,
+    positive,
+    icon,
+    bold,
+    className,
+    dataTest,
+    children,
+}) => (
+    <div
+        data-test={dataTest}
+        className={cx(className, {
+            neutral,
+            positive,
+            negative,
+            bold,
+        })}
+    >
+        {icon && <TagIcon>{icon}</TagIcon>}
+        {children}
+        <style jsx>
+            {`
+                div {
+                    padding: 5px 6px 3px;
+                    border-radius: 3px;
+                    background-color: ${colors.grey300};
+                    fill: ${colors.grey700};
+                    color: ${colors.grey900};
+                    max-width: 240px;
+                    display: inline-flex;
+                    font-size: 13px;
+                    line-height: 12px;
+                    height: 23px;
+                    overflow: hidden;
+                    white-space: nowrap;
+                    text-overflow: ellipsis;
+                }
+                .negative {
+                    background-color: ${colors.red100};
+                    fill: ${colors.red800};
+                    color: ${colors.red900};
+                }
+
+                .neutral {
+                    background-color: ${colors.blue100};
+                    fill: ${colors.blue800};
+                    color: ${colors.blue900};
+                }
+
+                .positive {
+                    background-color: ${colors.green100};
+                    fill: ${colors.green800};
+                    color: ${colors.green900};
+                }
+
+                .bold {
+                    font-weight: 700;
+                    background-color: ${colors.grey700};
+                    color: ${colors.white};
+                    fill: ${colors.white};
+                }
```

Comment:
I'm not 100% sold on the `bold` prop-name, because this is not only influencing the font-weight, but also the colours. This might be unexpected behaviour. Should we perhaps use a different name here, like `emphasized` which is more annoying to type but has a slightly more generic meaning than bold?

Reply:
My thinking is that `bold` is also a descriptor of the colour too. The color is bold, where the regular tag colours are muted/pastel. `Bold` as a prop-name is intended to refer to the usage (use this when you need to make a bold statement), rather than a descriptor of appearance. I'm not sure if this is frowned upon for prop names in general?

Reply:
Interesting... When doing web development and reading `bold` I immediately think of _a bold font_ (thicker characters) probably because of `font-weight: bold;`. But you are right, the actual definition is much broader: _having a strong, vivid, or clear appearance._ 

It is also an easy and short word to type, which is always good.

And thinking about it some more, my suggestion `emphasized` suffers from a similar problem, due to the `<em/>` tag.

Taken the above into account, I think that `bold` is probably fine.